synfig-studio/src/gui/resources/css/synfig.css

```diff
@@ -97,3 +97,8 @@
 #canvasview {
 	background: @theme_unfocused_bg_color;
 }
+
+#timetrack {
+	font-size: 11px;
```

Comment:
Using negation `:not(...)` would skip customizing column header fonts:
```css
#timetrack .view:not(header) { ... }
```

Reply:
I agree.

This way headers are not scaled and it lets users to set the desired size

Reply:
@ice0 ping

Reply:
Hi, guys!
Sorry for a late reply.
Something has changed since this commit, so I'm trying to find a new solution.
1) After rebase this is not working anymore.
2) `#timetrack .view:not(header) { ... }` this is not working for me. Maybe I am doing something wrong.
3)
>  `Why don't we use percentages instead?`. 

This is not work how it should be. I will show this in video below.

> Messaged the official gtk dev team [here](https://matrix.to/#/!aUhETchlgthwWVQzhi:matrix.org/$1670292462134719FmuqD:matrix.org?via=gnome.org&via=matrix.org&via=libera.chat) for solution

Unfortunately I can't see the answer.
![image](https://github.com/synfig/synfig/assets/5604544/0a4d968c-7d1f-4331-9432-02a4f26a3cad)


Video 1. Trying to use `:not` selector:

https://github.com/synfig/synfig/assets/5604544/5abbd693-c313-4a4d-9364-91bf3f8e4fd7


Video 2. font not changing in macOS:

https://github.com/synfig/synfig/assets/5604544/661674e7-73a3-4c2c-823e-434ebccbe803

Reply:
About the second video: it seems your build uses current code (parameter panel font is a fixed value in code)

Reply:
> About the second video: it seems your build uses current code (parameter panel font is a fixed value in code)

I don't know how you noticed this, but you're right! I forgot to cherry-pick changes to the current master before testing lol.

Reply:
I tested here too, and the `not` operator does not seem to work for this case.
And, as you said, the `header` node inherits the `view` :/

So, the only way to not affect the header is using two selectors:

- #parameters_panel
- #parameters_panel header

So we set the font-size in the first selector and the we revert it back in the second one.

```css
#parameters_panel {
  font-size: 70%;
}

#parameters_panel header {
  font-size: 143%; /* 1 / 0.70 */
}

```

So, what is the best option?